src/api/posts/src/storage.js

```diff
@@ -80,8 +80,9 @@ module.exports = {
     const invalidKeys = await getFeedKeysUsingScanStream(`${feedNamespace}*${invalidSuffix}`);
     return Promise.all(
       invalidKeys.map(async (key) => {
-        const reason = await redis.get(key);
+        let reason = await redis.get(key);
         const id = key.replace(feedNamespace, '').replace(invalidSuffix, '');
+        reason = reason.replace(/\n/g, ' ');
```

Comment:
Instead of making `reason` let vs. const, do this:

```
return {
  id,
  reason: reason.replace(/\n/g, ' '),
}
```

Don't modify or create variables when you can avoid it.